### ISSUE TITLE
Fix warning for CORS

### ISSUE THREAD

**Issue by rweigel** [hapi-server/verifier-nodejs#6]
The CORS settings are not provided in every response. Only the ones where the client asks for them.  Like Bernie suggested, I only send CORS response headers if the request header contains a non-empty "Origin" indicator. The validator is evidently not setting the "Origin" header in the request, so it won't get the CORS responses.

**Comment by berniegsfc** [comment]
The CORS specification https://www.w3.org/TR/access-control/ is pretty clear about syntax but I don't claim to understand all the use cases.  I've always considered testing with several browsers as necessary to access compliance.  And some browsers don't like to show you their pre-flight requests.  My SSC and CDAS web services currently work with all major browsers.  They only respond to an Origin header with Access-Control-Allow-Origin and Access-Control-Allow-Methods headers and to a Access-Control-Request-Headers with an Access-Control-Allow-Headers header.  Maybe they should do something different but this seems to be sufficient.  I think the hapi verifier should be consistent with most browsers but I can't be more specific than what I've observed with my testing.

**Comment by rweigel** [comment]
@berniegsfc what is the minimal header info that you need for your AJAX client to work? I think that it is

```
Access-Control-Allow-Origin: *
Access-Control-Allow-Methods: GET
```

**Comment by berniegsfc** [comment]
I just ran my hapi ajax client in chrome and ff with the network console open.  It looks like all that was required was
Access-Control-Allow-Origin: *
The Access-Control-Allow-Headers may only be required with the SSC and CDAS web services which are slightly more complicated (supporting POST of JSON and XML).  Or maybe it was for IE, Edge, or Safari which I didn't test today.  I gave up predicting what browsers want a long time ago.
